tests/UnitTests/EnabledTagHelperTests.cs

```diff
@@ -0,0 +1,70 @@
+﻿using Microsoft.AspNetCore.Razor.TagHelpers;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TagHelperPack;
+using Xunit;
+
+namespace UnitTests
+{
+    public class EnabledTagHelperTests
+    {
+        [Fact]
+        public void IsIntanceOf_TagHelper()
+        {
+            var helper = new EnabledTagHelper();
+
+            Assert.IsAssignableFrom<TagHelper>(helper);
+        }
+
+        [Fact]
+        public void IsEnabled_True_ByDefault()
+        {
+            var helper = new EnabledTagHelper();
+
+            Assert.True(helper.IsEnabled);
+        }
+
+        [Fact]
```

Comment:
Let's change this to a `[Theory]` that tests for each element the Tag Helper supports.

Reply:
Got it. I have to google Theory vs Fact really quick. But I think I got what you mean. Could you check my update?